evap/student/templates/student_vote.html

```diff
@@ -98,14 +111,14 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
                         <div class="card collapsible{% if not forloop.last %} mb-3{% endif %}">
                             <div class="card-header d-flex tab-row">
                                 <div class="me-auto">
-                                    <button class="collapse-toggle{% if errors_exist and not contributor_has_errors %} collapsed tab-selectable{% endif %} bg-transparent" data-bs-toggle="collapse"
+                                    <button class="collapse-toggle{% if general_errors_exist or contributor_errors_exist and not contributor_has_errors %} collapsed tab-selectable{% endif %} bg-transparent" data-bs-toggle="collapse"
```

Comment:
Django templating language follows the common operator precedence of "and" binding stronger than "or" ([docs](https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#boolean-operators)). You need parentheses around the `or` here.